packages/web/src/common/store/buy-audio/constants.ts

```diff
@@ -0,0 +1,40 @@
+export type JupiterTokenListing = {
```

Comment:
I might add a comment that these are copy/pasted from the results of getting the token listens via the Jupiter API. I figured they're not going to change, why query for them?